src/lib/api/discovery-service-api/discoveryServiceApi.ts

```diff
@@ -5,6 +5,8 @@ import { ApiResponseWrapper, wrapErrorCode, wrapSuccess } from 'lib/util/apiResp
 import { SpecificAssetId } from '@aas-core-works/aas-core3.0-typescript/types';
 import * as path from 'node:path';
 import ServiceReachable from 'test-utils/TestUtils';
+import Logger, { logResponseInfo } from 'lib/util/Logger';
```

Comment:
When importing the baseLogger it should be named in camelCase as it is an instance of the pino Logger. Also in all the other files of course. 
```suggestion
import logger, { logResponseInfo } from 'lib/util/Logger';
```